background.ts

```diff
@@ -84,6 +84,16 @@ Electron.app.whenReady().then(async() => {
     }
 
     buildApplicationMenu();
+    if (os.platform().startsWith('win')) {
```

Comment:
Is there any reason we don't do this on mac too?

Reply:
The Mac About box is populated from info in `electron-builder.yml`, but I guess we should go with the common code two keep the contents of the different supported systems in sync.